mindjunkies/accounts/models.py

```
@@ -13,6 +13,9 @@ class User(AbstractUser):
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 
     REQUIRED_FIELDS = ["email", "first_name", "last_name"]
+    is_teacher = models.BooleanField(default=False)
+    objects = UserManager()
```

Ruff (0.8.2)

Undefined name `UserManager`

(F821)